Makefile

```diff
@@ -290,7 +290,7 @@ clean-docker: ## Deletes the docker containers for local development.
 
 govet: ## Runs govet against all packages.
 	@echo Running GOVET
-	$(GO) vet $(GOFLAGS) $(TE_PACKAGES) || exit 1
+	$(GO) vet -shadow $(GOFLAGS) $(TE_PACKAGES) || exit 1
 
 ifeq ($(BUILD_ENTERPRISE_READY),true)
 	$(GO) vet $(GOFLAGS) $(EE_PACKAGES) || exit 1
```

Comment:
There are some shadowing issues with the enterprise code, though similarly not many. Would be good to get this resolved at some point as well :)

Reply:
Well, I don't have access to the enterprise code. But if I had, I would look into it.